packages/common/src/tools/Logger.ts

```diff
@@ -20,7 +21,11 @@ export interface LoggerOptions {
 export type LoggerParameters = Record<string, json>
 
 export class Logger {
-  constructor(private readonly options: LoggerOptions) {}
+  private readonly logThrottler: LogThrottler
+
+  constructor(private readonly options: LoggerOptions) {
+    this.logThrottler = new LogThrottler(5, 10000, 20000)
```

Comment:
make these parameters part of the logger options along side "throttling: boolean" to enable/disable this feature